src/index.md

```diff
@@ -321,9 +321,10 @@ It's not the reporter's responsibility to set a correct priority, but a qualifie
 A few things to keep in mind when filing a new bug:
 
 * Before filing a bug, verify that there isn't already a bug filed for this issue.
-* If you suspect that the bug is a vulnerability, don't file a JBS issue. Instead send your bug report to [vuln-dev@openjdk.java.net](mailto:vuln-dev@openjdk.java.net)
+  * Search [JBS](https://bugs.openjdk.java.net/) for things like the name of the failing test, assert messages, the name of the source code file where a crash occurred etc.
+* If you suspect that the bug is a vulnerability, don't file a JBS issue. Instead send your bug report to [vuln-report@openjdk.java.net](mailto:vuln-report@openjdk.java.net). See [OpenJDK Vulnerabilities](https://openjdk.java.net/groups/vulnerability/report) for more information.
```

Comment:
This looks fine but maybe add a note to say that the issue shouldn't be sent to any of the other OpenJDK mailing lists.

Reply:
Good point! Added a note on that.